From a33d8c40e9e56597f68e83be3336036992996738 Mon Sep 17 00:00:00 2001
From: Matthew Miller <mattdm@mattdm.org>
Date: Thu, 8 Apr 2021 20:21:39 -0400
Subject: [PATCH] Fix doc building with newer yelp

This adds `-i`, because otherwise it will complain about missing files (which
we copy into place later), and adds a trailing slash to the output directory,
because `yelp-build` now puts the files into the wrong place otherwise.
---
 doc/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/doc/Makefile.am b/doc/Makefile.am
index 3d044bd8..78b96233 100644
--- a/doc/Makefile.am
+++ b/doc/Makefile.am
@@ -7,7 +7,7 @@ EXTRA_DIST = docbook2html.sh docbook icons
 html/GuideIndex.html: docbook/GuideIndex.xml
 	rm -rf html; mkdir html; cp $(srcdir)/icons/* html/
 	if [ -x "$(GNOME_DOC_TOOL)" ]; then \
-		"$(GNOME_DOC_TOOL)" html -o html $(srcdir)/docbook/GuideIndex.xml ; \
+		"$(GNOME_DOC_TOOL)" html -i -o html/ $(srcdir)/docbook/GuideIndex.xml ; \
 	else \
 		echo "yelp-tools not found, html is not built" ; \
 	fi
